File: Backjoon/5585_거스름돈/첫번째 시도_성공.c
#include <iostream>

using namespace std;
int main() {
	int input,output = 0;
	int coin[6] = { 500,100,50,10,5,1 };
	cin >> input;
	input = 1000 - input;
	for (int i = 0; i < 6; i++) {
		while (input / coin[i] != 0) {
			output += input / coin[i];
			input %= coin[i];
		}
	}
	cout << output;
	return 0;
}

File: 5585_거스름돈/첫번째 시도_성공.py
coin = [500,100,50,10,5,1]
input_num = int(input())
output = 0
input_num = 1000-input_num
for i in range(6):
    while (input_num//coin[i]) != 0:
        output += input_num//coin[i]
        input_num %= coin[i]
        
print(output)
